iOS/WindowsAzureMessaging/WindowsAzureMessaging.xcodeproj/project.pbxproj

```diff
@@ -408,6 +408,8 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				IPHONEOS_DEPLOYMENT_TARGET = 5.1.1;
+				OTHER_CFLAGS = "-fembed-bitcode";
```

Comment:
This won't be enough.

You need to use two different flags. For simulator and device debug builds, you'll need `-fembed-bitcode-marker`, for release builds, use -fembed-bitcode.

Reply:
Some more context, `fembed-bitcode-marker` will "simulate" the bitcode, that's what we use for debugging since bitcode takes compilation time. We only enable full bitcode `-fembed-bitcode` on release builds.

Reply:
@ElektrojungeMS is also mentioning simulator vs device. Simulators just don't need bitcode since bitcode is only useful to packages submitted to the store and those packages don't contain the simulator slice.